### ISSUE TITLE
Links should be visually distinct without relying on color

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7391]
We should consider and discuss how this can look without making our site look messy.

**Comment by sigurdm** [comment]
The standard way here is to underline all links...

**Comment by sigurdm** [comment]
We could also consider underlining on hover only... seems to be good enough for google search results.

**Comment by sigurdm** [comment]
https://dequeuniversity.com/rules/axe/4.8/link-in-text-block
